src/__tests__/MetaOptions.js

```diff
@@ -0,0 +1,24 @@
+import MetaOptions from '../MetaOptions'
+
+describe('MetaOptions', () => {
+  describe('Initialisation', () => {
+    it('is valid with a valida options object', () => {
+      const options = { bodyKeyCase: 'SNAKE_CASE' }
+      const metaOptions = new MetaOptions(options)
+      expect(metaOptions).toBeTruthy()
+    })
+
+    it('throws an error if no options are passed', () => {
+      expect(() => new MetaOptions()).toThrow()
+    })
+
+    it('throws an error if bodyKeyCase is not present in the options', () => {
```

Comment:
Why does it throw an error if there's a default value? 🤔

Reply:
There is no default value present in `MetaOptions`. Default values are handled by the `Snuffles` class.

Reply:
Ah, sorry, I was in the wrong class.